cmd/kev/cmd/init.go

```diff
@@ -118,25 +119,43 @@ func runInitCmd(cmd *cobra.Command, _ []string) error {
 	if skaffold {
 		skaffoldManifestPath := path.Join(workingDirAbs, kev.SkaffoldFileName)
 
-		skaffoldManifest, err := kev.PrepareForSkaffold(manifest, skaffoldManifestPath, envs)
-		if err != nil {
-			return displayInitError(err)
-		}
+		// set skaffold path in kev manifest
+		manifest.Skaffold = kev.SkaffoldFileName
 
-		// don't override existing skaffold.yaml
 		if manifestExistsForPath(skaffoldManifestPath) {
-			results = append(results, skippableFile{
-				FileName: kev.SkaffoldFileName,
-				Skipped:  true,
-			})
-		} else if skaffoldManifest != nil {
-			if err := writeTo(kev.SkaffoldFileName, skaffoldManifest); err != nil {
+			// skaffold manifest already present - add additional profiles to it!
+			// Note: kev will skip profiles with names matching those of existing
+			// profile names defined in skaffold to avoid profile "hijack".
+
+			updatedSkaffold, err := kev.AddProfiles(skaffoldManifestPath, envs, true)
+			if err != nil {
+				return displayInitError(err)
+			}
+			if err := writeTo(skaffoldManifestPath, updatedSkaffold); err != nil {
 				return displayInitError(err)
 			}
 
 			results = append(results, skippableFile{
 				FileName: kev.SkaffoldFileName,
+				Updated:  true,
 			})
+
+		} else {
+
+			skaffoldManifest, err := kev.PrepareForSkaffold(envs)
+			if err != nil {
+				return displayInitError(err)
+			}
+
+			if skaffoldManifest != nil {
```

Comment:
Not sure why this check required, as the lines before already do the work.

Reply:
old artefact. removed.